client/Dockerfile.fedora

```diff
@@ -1,11 +1,10 @@
-FROM fedora:25
+FROM fedora:26
 MAINTAINER Luiz Carvalho <lucarval@redhat.com>
 
 ARG UPDATES_TESTING
 
 RUN dnf update -y && dnf install -y dnf-plugins-core && \
-    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing; fi && \
-    dnf -y update && \
+    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing && dnf -y update; fi && \
```

Comment:
Parent images don't get updated fast enough - for instance, fedora's update cycle is two weeks, for rhel its even more complicated.

Our only worry is unnecessary update of k-cb, atomic-reactor and osbs-client - this can be solved by setting up proper repos - or updating dockerfiles if required: `atomic-reactor == 1.6.24-1`